frontend/src/modules/Sensitivity/view.tsx

```diff
@@ -167,6 +194,7 @@ export const view = ({ moduleContext, workbenchSession, workbenchServices }: Mod
                     </div>
                 )}
             </div>
+            <div className="absolute top-10 left-5 italic text-pink-400">(rc={renderCount.current})</div>
```

Comment:
Should we add a general debug info in `dev` mode for this for all modules? Some sort of footer overlay with num renders, last/max/min/avg render time? This line should be removed for demo. Moreover, further up, the main div takes 100% of its parents height and is assigned the ref that determines the plot's height although there are buttons added on top. The layout does not work here and I would vote for a more thorough refactoring of this module (independent of this PR of course).

Reply:
Didn't look into the layout of this module at all. It certainly needs more love both wrt layout and implementation.
Helper functionality for monitoring render count and render time could be useful at some point.